packages/suite/src/components/suite/modals/ReduxModal/ConfirmAddressModal.tsx

```diff
@@ -39,16 +39,16 @@ export const ConfirmAddressModal = ({ addressPath, value, ...props }: ConfirmAdd
 
     const getHeading = () => {
         if (modalCryptoId) {
-            const coinSymbol = cryptoIdToCoinSymbol(modalCryptoId)?.toUpperCase();
-            const symbol = cryptoIdToSymbol(modalCryptoId)?.toUpperCase();
+            const coinSymbol = cryptoIdToCoinSymbol(modalCryptoId)?.toLowerCase();
+            const symbol = cryptoIdToSymbol(modalCryptoId);
 
             if (symbol && coinSymbol !== symbol) {
                 return (
                     <Translation
                         id="TR_ADDRESS_MODAL_TITLE_EXCHANGE"
                         values={{
-                            networkSymbol: symbol,
-                            networkCurrencyName: coinSymbol,
+                            networkSymbol: getNetworkDisplaySymbol(symbol),
+                            networkCurrencyName: coinSymbol?.toUpperCase(),
```

Comment:
Because of L2s, there should be `on BSC network` or `on BNB Smart Chain network` so use network name or network symbol. Probably name is better?
<img width="853" alt="Screenshot 2025-01-03 at 12 44 00" src="https://github.com/user-attachments/assets/fb8309de-3b85-4400-ad30-a41d59c1472b" />